tests/features/multilingual-aliases.feature

```diff
@@ -23,16 +24,47 @@ Feature: Webtools Analytics multilingual aliases
     And I fill in "Section" with "multilingual"
     And I check the box "Match on path alias for site default language."
     And I fill in "Regex" with "/news/"
-    When I press "Save"
+    Then I press "Save"
     # Check the rule applies
-    Given I am on "/admin/config"
+    When I am on "/admin/config"
     Then the page analytics json should contain the parameter "siteSection" with the value "multilingual"
-    Given I am on "/news"
+    When I am on "/news"
     Then the page analytics json should contain the parameter "siteSection" with the value "multilingual"
-    Given I am on "/fr/nouvelles"
+    When I am on "/fr/nouvelles"
     Then the page analytics json should contain the parameter "siteSection" with the value "multilingual"
-    Given I am on "/nl/nieuws"
+    When I am on "/nl/nieuws"
     Then the page analytics json should contain the parameter "siteSection" with the value "multilingual"
-    Given I am on "/admin"
+    When I am on "/admin"
     Then the page analytics json should not contain the parameter "siteSection"
 
+  @cleanup:webtools_analytics_rule @cleanup-aliases
+  Scenario: Make sure that Webtools Analytics rules work without checked "Match on path alias for site default language"
```

Comment:
`Make sure that Webtools Analytics rules work without matching the alias in the site default language`

Reply:
Done